### DISCUSSION TITLE
Question non-convex into convex

### DISCUSSION THREAD

**Discussion by michaldengusiak** [NetTopologySuite/NetTopologySuite#493] [Q&A]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of NetTopologySuite
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/NetTopologySuite/NetTopologySuite/issues) and [closed](https://github.com/NetTopologySuite/NetTopologySuite/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
_(A description of the bug or feature)_
Do you have any method to split/divide non-convex(concave)  polygon into convex?
I want to keep polygon holes... in case there are included
ie. 
![image](https://user-images.githubusercontent.com/26113670/92123421-89ec7680-edf4-11ea-9a28-86cf3379a027.png)

### Steps to Reproduce
_(List of steps, sample code, failing test or link to a project that reproduces the behavior)_

### System Configuration
Tell us about the environment where you are experiencing the bug:
- NetTopologySuite version:
- Other NetTopologySuite packages and versions:
- Environment (Operating system, version and so on):
- .NET Framework version:
- Additional information:

_Thanks for reporting the issue to NetTopologySuite!_

**Comment by FObermaier** [comment]
I'm sorry, there is no such functionality in NTS.
